adaptive_navigation/test/adaptive_navigation_test.dart

```diff
@@ -1,41 +0,0 @@
-import 'package:flutter/material.dart';
```

Comment:
Why is this file being deleted? It seems unrelated

Reply:
Unchanged this